starfish/test/full_pipelines/api/test_dartfish.py

```diff
@@ -146,7 +146,7 @@ def test_dartfish_pipeline_cropped_data():
 
     # verify number of spots detected
     spots_passing_filters = spot_intensities[Features.PASSES_THRESHOLDS].sum()
-    assert spots_passing_filters == 54
+    assert spots_passing_filters == 53
```

Comment:
say what?

Reply:
This reverts a previous value change, so I was not concerned about it:

5194045